src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -11,18 +11,56 @@ import {
 } from '@angular/core';
 import { FieldDragDropService, DropFieldData } from '../service';
 import { FieldType } from '../models';
-import { DotCMSContentTypeField, DotCMSContentTypeLayoutRow } from 'dotcms-models';
+import {
+    DotCMSContentTypeField,
+    DotCMSContentTypeLayoutRow,
+    DotCMSContentTypeLayoutColumn
+} from 'dotcms-models';
 import { ContentTypeFieldsPropertiesFormComponent } from '../content-type-fields-properties-form';
 import { DotMessageService } from '@services/dot-messages-service';
-import { FieldUtil } from '../util/field-util';
+import { FieldUtil, COLUMN_FIELD, COLUMN_BREAK_FIELD } from '../util/field-util';
 import { FieldPropertyService } from '../service/field-properties.service';
 import { DotDialogActions } from '@components/dot-dialog/dot-dialog.component';
 import { DotEventsService } from '@services/dot-events/dot-events.service';
-import { takeUntil, take  } from 'rxjs/operators';
+import { takeUntil, take } from 'rxjs/operators';
 import { Subject } from 'rxjs';
 import { DotLoadingIndicatorService } from '@components/_common/iframe/dot-loading-indicator/dot-loading-indicator.service';
 import * as _ from 'lodash';
 
+const splitRows = (fieldRows: DotCMSContentTypeLayoutRow[]): DotCMSContentTypeLayoutRow[] => {
+    return fieldRows.map((row) => {
+        return {
+            ...row,
+            columns: row.columns.reduce(
+                (acc: DotCMSContentTypeLayoutColumn[], current: DotCMSContentTypeLayoutColumn) => {
+                    const indexOfBreak = current.fields.findIndex((item) => {
+                        return item.clazz === COLUMN_BREAK_FIELD.clazz;
+                    });
+                    if (indexOfBreak > -1) {
```

Comment:
it allways should be > -1

https://github.com/dotCMS/core-web/pull/1093/files#diff-f55ac1869014ff27c1aad3262919eb93R148

Reply:
Are different intentions, one of the dropped on when we checking where was drop in the dragula model.